recipe/meta.yaml

```diff
@@ -28,7 +28,11 @@ requirements:
     - colorama >=0.3.9,<1
     - click >=6.7,<9
     - email_validator >=1.0.5,<2
-    - flask >=0.12,<2
+    # an incompatibility with itsdangerous is fixed with later flask versions
+    # - flask >=0.12,<2
+    - flask >=1.1.4,<2
+    # needed because flask <2 depends on jinja2 <3, which is not compabibel with the lastest markupsafe
+    - markupsafe <2
```

Comment:
I'm trying to fix this one upstream in the jinja2 feedstock but it could easily still be the case that the solver pulls in the previous builds without this constraint.  Because of that and also in order to make this release happen sooner rather than later, I'm going to leave this in.